src/webgpu/api/operation/buffers/map_ArrayBuffer.spec.ts

```diff
@@ -78,4 +83,8 @@ g.test('postMessage')
 
     buf.unmap();
     t.expect(ab1.byteLength === 0, 'after unmap, ab1 should be detached');
+    if (transfer) {
```

Comment:
nit: I think we can/should test this in both cases, both `transfer=true` and `false`.

Reply:
with transfer=false it should work though because it will get copied?
oh - do you mean to test again that it does not throw an exception?

Reply:
I mean, regardless of whether the test case was for transfer=true or transfer=false, still test at the end that transferring is invalid

Reply:
Oh I see - I understand now. Done